src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs

```diff
@@ -178,10 +178,12 @@ public AkkaProtocolStressTest() : base(AkkaProtocolStressTestConfig)
 
         #region Tests
 
-        [Fact(Skip = "Still have intermittent timeouts")]
+        [Fact()]
         public void AkkaProtocolTransport_must_guarantee_at_most_once_delivery_and_message_ordering_despite_packet_loss()
```

Comment:
This spec may be racy as reported by @JeffCyr, but I'd like to keep it on so we can capture some record of that in the logs.